connection.go

```diff
@@ -182,6 +182,12 @@ func (c *Connection) Q() *Query {
 	return Q(c)
 }
 
+// eagerDisabled disables eager mode for current connection.
+func (c *Connection) eagerDisabled() {
```

Comment:
@larrymjordan should this be called something like "disableEager" ?

Reply:
@paganotoni yes, that makes sense.